server/venueless/core/migrations/0065_alter_auditlog_data_alter_chatevent_content_and_more.py

```diff
@@ -0,0 +1,120 @@
+# Generated by Django 4.2.13 on 2024-07-16 08:49
+
+from django.db import migrations, models
+import django.db.models.deletion
+import venueless.core.models.room
+import venueless.core.models.world
+import venueless.core.utils.json
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0064_auto_20230805_1737"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="auditlog",

```

Comment:
Why are there migrations relatd to `AuditLog` model? I don't see changes of `AuditLog` model in this PR.